### PR TITLE
added vlanindex package

### PR DESCRIPTION
**Pull request by henderiw**
provisions a vlanindex per cluster for VLAN claims

### CODE REVIEW

File: nephio-workload-cluster/pv-vlanindex.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+metadata:
+  name: example-repo
+spec:
+  annotations:
+    approval.nephio.org/policy: initial
+    approval.nephio.org/delay: 2m
+  upstream:
+    package: vlanindex
+    repo: nephio-example-packages
+    revision: v3
```
Comment (johnbelamaric) [inline nephio-workload-cluster/pv-vlanindex.yaml]: Should be v1, right?

**Review by johnbelamaric — COMMENTED** [review]
Looks like one tweak needed.

File: nephio-workload-cluster/pv-vlanindex.yaml (hunk 1 above)
Comment (henderiw) [reply nephio-workload-cluster/pv-vlanindex.yaml]: fixed

**Pull request merged**